### PR TITLE
Add 6 aliases to fix dead links

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Add 6 aliases to fix dead links.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Review by ran-huang — APPROVED** [review]

File: en/backup-and-restore-using-helm-charts.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Backup and Restore Using Helm Charts
 summary: Learn how to back up and restore data of the TiDB cluster in Kubernetes using Helm charts.
-aliases: ['/docs/tidb-in-kubernetes/stable/backup-and-restore-using-helm-charts/','/docs/tidb-in-kubernetes/v1.1/backup-and-restore-using-helm-charts/','/docs/dev/tidb-in-kubernetes/maintain/backup-and-restore/','/docs/v3.0/tidb-in-kubernetes/maintain/backup-and-restore/','/docs/v3.1/tidb-in-kubernetes/maintain/backup-and-restore/','/docs/v3.0/how-to/maintain/tidb-in-kubernetes/backup-and-restore/','/docs-cn/dev/tidb-in-kubernetes/maintain/backup-and-restore/restore/']
+aliases: ['/docs/tidb-in-kubernetes/stable/backup-and-restore-using-helm-charts/','/docs/tidb-in-kubernetes/v1.1/backup-and-restore-using-helm-charts/','/docs/dev/tidb-in-kubernetes/maintain/backup-and-restore/','/docs/v3.0/tidb-in-kubernetes/maintain/backup-and-restore/','/docs/v3.1/tidb-in-kubernetes/maintain/backup-and-restore/','/docs/v3.0/how-to/maintain/tidb-in-kubernetes/backup-and-restore/','/docs-cn/dev/tidb-in-kubernetes/maintain/backup-and-restore/restore/','/docs/dev/tidb-in-kubernetes/maitain/backup-and-restore/']
```
Comment (yikeke) [inline en/backup-and-restore-using-helm-charts.md:4]: ,'/docs-cn/dev/tidb-in-kubernetes/maintain/backup-and-restore/restore/','/docs/dev/tidb-in-kubernetes/maitain/backup-and-restore/'] -> ,'/docs/dev/tidb-in-kubernetes/maitain/backup-and-restore/']

Comment (yikeke) [reply en/backup-and-restore-using-helm-charts.md:4]: @TomShawn

Comment (TomShawn) [reply en/backup-and-restore-using-helm-charts.md:4]: Thanks for pointing out. I'll create a PR to fix today.

Comment (TomShawn) [reply en/backup-and-restore-using-helm-charts.md:4]: Fixed in https://github.com/pingcap/docs-tidb-operator/pull/821/files#diff-3d0c885b197d038e95373e699a61202e8efaeb924d6b5b498210ec6b99172364R4.

**Pull request merged**